elliott/elliottlib/cli/find_bugs_sweep_cli.py

```diff
@@ -350,6 +354,7 @@ def categorize_bugs_by_type(
     operator_bundle_advisory: str = "metadata",
     permissive=False,
     major_version: int = 4,
+    minor_version: int = 0,
```

Comment:
i would recommend to not have default values here, so 
```suggestion
    major_version: int,
    minor_version: int,
```

Reply:
Done